src/iso19111/crs.cpp

```diff
@@ -1169,6 +1169,12 @@ CRSNNPtr CRS::normalizeForVisualization() const {
         }
     }
 
+    if (const BoundCRS *boundCRS = dynamic_cast<const BoundCRS *>(this)) {
+        auto baseNormCRS = boundCRS->baseCRS()->normalizeForVisualization();
+        return BoundCRS::create(baseNormCRS, boundCRS->hubCRS(),
+                                boundCRS->transformation());
+    }
```

Comment:
Thanks for this fix. You forgot to simplify the code that calls this function assuming it doesn't handle BoundCRS.

Reply:
Ignore this comment, I got confused by other code base.